src/js/views/forms/form/form.scss

```diff
@@ -30,6 +30,24 @@
   }
 }
 
+.form__widgets {
+  background-color: $white;
+  border: 1px solid $black-90;
+  border-radius: 4px;
+  margin-bottom: 16px;
+  padding: 8px 8px 0 8px;
+
+  > div:first-of-type {
```

Comment:
We discussed possibly replacing this with a class name, but I couldn't figure out how to do that. This works, but doesn't feel like the cleanest solution.

 Maybe you could find a better solution instead of what I did.

This is the current HTML structure of the form widgets container:

```html
<div class="form__widgets flex">
  <div> <! –– automatically added somewhere by the widgets view? ––>
    <div class="form__widgets-section">
      <! –– widget heading and item ––>
    </div>
    <div class="form__widgets-section">
      <! –– widget heading and item ––>
    </div>
  </div>
</div>
```

That empty `<div>` is added somewhere. By the widgets view? I'm not sure. But I couldn't find a place to attach a class name to it.